src/rust/bitbox02-rust/src/hww/api/cardano/sign_transaction/certificates.rs

```diff
@@ -88,6 +88,57 @@ pub async fn verify<'a>(
                 })
                 .await?;
             }
+            VoteDelegation(certificate::VoteDelegation {
+                keypath,
+                r#type,
+                drep_credhash,
+            }) => {
+                validate_address_shelley_stake(keypath, Some(bip44_account))?;
+                signing_keypaths.push(keypath);
+                let drep_type_name =  match certificate::vote_delegation::CardanoDRepType::try_from(*r#type)? {
+                   certificate::vote_delegation::CardanoDRepType::KeyHash => "Key Hash",
+                   certificate::vote_delegation::CardanoDRepType::ScriptHash => "Script Hash",
+                   certificate::vote_delegation::CardanoDRepType::AlwaysAbstain => "Always Abstain",
+                   certificate::vote_delegation::CardanoDRepType::AlwaysNoConfidence => "Always No Confidence",
+                };
+                confirm::confirm(&confirm::Params {
+                        title: params.name,
+                        body: &format!(
+                            "Delegate voting for account #{}?",
+                            keypath[2] + 1 - HARDENED,
+                        ),
+                        scrollable: true,
+                        accept_is_nextarrow: true,
+                        ..Default::default()
+                    })
+                    .await?;
+                match drep_credhash {
+                    Some(hash) => { confirm::confirm(&confirm::Params {
+                        title: params.name,
+                        body: &format!(
+                            "to type #{} and drep #{}?",
```

Comment:
why the hashtag `#`?

Reply:
I just now realized it was there to signify number in the message so I removed them.